css/css-color/oklch-011.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: OKLab and OKLCH</title>
+<link rel="author" title="James Stuckey Weber" href="mailto:james@jamessw.com">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#specifying-oklab-oklch">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#predefined-to-lab-oklab">
+<link rel="match" href="greensquare-ref.html">
+<meta name="assert" content="oklch() specifying C with percentage">
+<style>
+    .test { background-color: red; width: 12em; height: 12em; }
+    .test { background-color: oklch(51.975% 44.215% 142.495); } /* green (sRGB #008000) converted to OKLab */
```

Comment:
converted to Oklch :)
also I get 44.225% for chroma but the difference is not visible

Reply:
Good catch!

For the percentage- I divided the `0.17686` given in `oklch-001.html` by `0.4` to normalize into the reference range , and got `0.44215` or `44.215%`.  

I did some checking in Safari, and you're correct that 44.225% and 44.215% are both displayed as `color(srgb 0 0.5019 0.0053)`. This made me curious about which digits are actually significant, and 44.2% and 44% are also both identical. Given this, I updated the test to 44%, but am open to whatever precision makes the most sense here.